python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -248,6 +371,157 @@ def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fie
                                                                     for column in self.active_columns if column[2] is not None]))
                                           for pointing in results]
 
-
         return obs_output
 
+    @staticmethod
+    def ObservationMetaDataForPointing(OpSimPointingRecord, columnMap,
```

Comment:
I think we have two standards developing.  The astrometry routines that convert X into Y are named ```xFromY()``` (e.g. ```pixelCoordsFromRaDec```).  The sims_catUtils methods are named ```xForY()```.  I prefer 'From'.  It just sounds more active to me.  What is the argument for using 'For'?  (I feel like we should try to standardize this so users can develop an intuition when hunting for methods in the sims stack).